Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "nrfxlib-sys"
-version = "2.4.2"
+version = "2.7.0"
```

Comment:
This is technically not correct since this is a breaking change update.

Could you put in the readme in the 'Using' section that this crate does not follow semver and that you should pin the crate to an exact version? (Oh and update the version mentioned in the readme)

At the same time you can add to the changelog in the readme.

Reply:
I looked at the version history and just guessed that this crate's version tracked the nordic library versions.